docs/helm-chart-checks.md

```diff
@@ -131,18 +131,34 @@ This section provides help on the basic usage of Helm chart checks with the podm
 
 ### Cluster Config
 
-Configuration of the `chart-testing` check can be achieved through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
+Configuration of the `chart-testing` check can be achieved in one of two ways: 
+
+1. Through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
     ```text
     $ chart-verifier                                                 \
         verify                                                       \
         --enable chart-testing                                       \
-        --set chartTesting.buildId=${BUILD_ID}                       \
-        --set chartTesting.upgrade=true                              \
-        --set chartTesting.skipMissingValues=true                    \
-        --set chartTesting.namespace=${NAMESPACE}                    \
-        --set chartTesting.releaseLabel="app.kubernetes.io/instance" \
+        --set chart-testing.buildId=${BUILD_ID}                       \
+        --set chart-testing.upgrade=true                              \
+        --set chart-testing.skipMissingValues=true                    \
+        --set chart-testing.namespace=${NAMESPACE}                    \
+        --set chart-testing.releaseLabel="app.kubernetes.io/instance" \
         some-chart.tgz
     ```
+1. Through the `--set-values` command line option, where a YAML file can be used to configure the check. For example for a config.yaml file content:
+   ```text
+    chart-testing:
+        buildId: <BUILD_ID>
+        upgrade: true
+        skipMissingValues: true
+        namespace: <NAMESPACE>
+        releaseLabel: "app.kubernetes.io/instance"
+    ```
+
+    Once the file is persisted, a chart can be verified:
```

Comment:
Question: When you say "Once the file ..", Which file do you mean?

Suggestion: Put a note saying: "Wait for the file to persist". And then, add after the Note: "After the file is persisted, you can verify a chart for the 'chart-testing' check configuration as follows:

Reply:
See above.

Reply:
@mmulholla Please see my comments above.